### ISSUE TITLE
Changing the directory orientation

### ISSUE THREAD

**Issue by Unalloyedcube** [blazeeboy/dirtree#7]
Is it possible to change the orientation of how the children expand? I have a large list that expands left to right, but would love for that to be top to bottom like an org chart.

**Comment by Unalloyedcube** [comment]
I was able to flip the orientation, but now the labels/names are not creating enough space between each one. Any suggestions?

**Comment by emad-elsaid** [comment]
Yeah that was one of the reason I didn't have that template, I'm not sure how to have space for the file name without colliding with the siblings, can you submit the template you worked on maybe I can help?
try to put it on jsfiddle so we can present results to each other
